lib/tests/streamlit/elements/camera_input_test.py

```diff
@@ -26,10 +29,37 @@ def test_just_label(self):
 
         c = self.get_delta_from_queue().new_element.camera_input
         self.assertEqual(c.label, "the label")
+        self.assertEqual(
+            c.label_visibility.value,
+            LabelVisibilityMessage.LabelVisibilityOptions.VISIBLE,
+        )
 
     def test_help_tooltip(self):
-        """Test that it can be called using a string for type parameter."""
+        """Test that it can be called with help parameter."""
         st.camera_input("the label", help="help_label")
 
         c = self.get_delta_from_queue().new_element.camera_input
         self.assertEqual(c.help, "help_label")
+
+    @parameterized.expand(
+        [
+            ("visible", LabelVisibilityMessage.LabelVisibilityOptions.VISIBLE),
+            ("hidden", LabelVisibilityMessage.LabelVisibilityOptions.HIDDEN),
+            ("collapsed", LabelVisibilityMessage.LabelVisibilityOptions.COLLAPSED),
+        ]
+    )
+    def test_label_visibility(self, label_visibility_value, proto_value):
+        """Test that it can be called with label_visibility parameter."""
+        st.camera_input("the label", label_visibility=label_visibility_value)
+
+        c = self.get_delta_from_queue().new_element.camera_input
+        self.assertEqual(c.label_visibility.value, proto_value)
+
```

Comment:
Would it be possible to write a test for the 3rd scenario, where `label="` and a warning is raised?